src/sentry/web/forms/accounts.py

```diff
@@ -321,442 +318,6 @@ def clean_password(self):
-        ],
-        help_text=_(
-            "This will be automatically set as your subscription preference when you create or join a project. It has no effect on existing projects."
-        ),
-        required=False,
-    )
-
-    self_notifications = forms.BooleanField(
-        label=_("Receive notifications about my own activity"),
-        help_text=_(
-            "Enable this if you wish to receive emails for your own actions, as well as others."
-        ),
-        required=False,
-    )
-
-    self_assign_issue = forms.BooleanField(
-        label=_("Claim unassigned issues when resolving them"),
-        help_text=_(
-            "When enabled, you'll automatically be assigned to unassigned issues when marking them as resolved."
-        ),
-        required=False,
-    )
-
-    def __init__(self, user, *args, **kwargs):
-        self.user = user
-        super(NotificationSettingsForm, self).__init__(*args, **kwargs)
-
-        self.fields["subscribe_by_default"].initial = (
-            UserOption.objects.get_value(user=self.user, key="subscribe_by_default", default="1")
-            == "1"
-        )
-
-        self.fields["workflow_notifications"].initial = UserOption.objects.get_value(
-            user=self.user,
-            key="workflow:notifications",
-            default=UserOptionValue.participating_only,
-            project=None,
-        )
-
-        self.fields["self_notifications"].initial = (
-            UserOption.objects.get_value(user=self.user, key="self_notifications", default="0")
-            == "1"
-        )
-
-        self.fields["self_assign_issue"].initial = (
-            UserOption.objects.get_value(user=self.user, key="self_assign_issue", default="0")
-            == "1"
-        )
-
-    def get_title(self):
-        return "General"
-
-    def save(self):
-        UserOption.objects.set_value(
-            user=self.user,
-            key="subscribe_by_default",
-            value="1" if self.cleaned_data["subscribe_by_default"] else "0",
-        )
-
-        UserOption.objects.set_value(
-            user=self.user,
-            key="self_notifications",
-            value="1" if self.cleaned_data["self_notifications"] else "0",
-        )
-
-        UserOption.objects.set_value(
-            user=self.user,
-            key="self_assign_issue",
-            value="1" if self.cleaned_data["self_assign_issue"] else "0",
-        )
-
-        workflow_notifications_value = self.cleaned_data.get("workflow_notifications")
-        if not workflow_notifications_value:
-            UserOption.objects.unset_value(
-                user=self.user, key="workflow:notifications", project=None
-            )
-        else:
-            UserOption.objects.set_value(
-                user=self.user,
-                key="workflow:notifications",
-                value=workflow_notifications_value,
-                project=None,
-            )
-
-
-class ProjectEmailOptionsForm(forms.Form):
```

Comment:
Why did these forms get removed if you're enabling a faceting option?

Reply:
I don't see anything being removed in the commit 
https://github.com/getsentry/sentry/commit/691dd67778e14751e109345fea03a985502b6711